src/core/Akka.Remote/Endpoint.cs

```diff
@@ -171,9 +173,9 @@ internal interface IAssociationProblem { }
     /// <summary>
     /// INTERNAL API
     /// </summary>
-    internal sealed class ShutDownAssociationException : EndpointException, IAssociationProblem
+    internal sealed class ShutDownAssociation : EndpointException, IAssociationProblem
```

Comment:
Changed the name of this exception to match the JVM (addressed some earlier comments by @rogeralsing)